lib/test/unit/assertions.rb

```diff
@@ -970,21 +1013,26 @@ def assert_not_in_delta(expected_float, actual_float, delta=0.001, message="")
       private
       def _assert_in_delta_validate_arguments(expected_float,
                                               actual_float,
-                                              delta)
-        {
-          expected_float => "first float",
-          actual_float => "second float",
-          delta => "delta"
-        }.each do |float, name|
-          assert_respond_to(float, :to_f,
-                            "The arguments must respond to to_f; " +
-                            "the #{name} did not")
+                                              delta,
+                                              type_conversion: true)
```

Comment:
@kou Because of the negative check, I introduced `type_conversion:` keyword argument to tell the necessity of `to_f` check.